src/client/app/App.tsx

```
@@ -277,117 +312,307 @@ const LandingHero = ({
+                                }}
+                                onMouseEnter={(e) => {
+                                    const el = e.currentTarget;
+                                    el.style.border = '1px solid rgba(0,229,255,0.4)';
+                                    el.style.boxShadow = '0 0 30px rgba(0,229,255,0.1), 0 8px 30px rgba(0,0,0,0.6)';
+                                }}
+                                onMouseLeave={(e) => {
+                                    const el = e.currentTarget;
+                                    el.style.border = '1px solid rgba(0,229,255,0.1)';
+                                    el.style.boxShadow = '0 4px 20px rgba(0,0,0,0.5)';
+                                }}
+                            >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)